lib/validator/validator.go

```diff
@@ -178,6 +178,10 @@ func validatePort(v *validator.Validate, structLevel *validator.StructLevel) {
 	if p.MinPort > p.MaxPort {
 		structLevel.ReportError(reflect.ValueOf(p.MaxPort), "Port", "port", "port range invalid")
 	}
+
+	if p.MinPort < 1 || p.MaxPort < 1 {
```

Comment:
This should (I know, I know) probably have a comment which says we don't need to check the upper bound because we use a uint16.